### PR TITLE
Clarify meaning in csg_tools

### PR DESCRIPTION
**Pull request by corrigentia**
Properly state the definition of the terms used; minor tweaks I had missed previously.

### CODE REVIEW

File: tutorials/3d/csg_tools.rst [hunk 1]
```diff
@@ -158,7 +158,7 @@ Create a ``CSGCylinder``, enable the ``cone`` option and make it the stand. Crea
 
 .. image:: img/csg_lamp_pole_stand.png
 
-We will use ``CSGPolygon`` for the lampshade. Use ``Spin`` mode for the ``CSGPolygon`` and draw a trapezoid in ``Front View`` (numeric keypad 1); this shape will extrude around the origin and form the lampshade.
+We will use a ``CSGPolygon`` for the lampshade. Use the ``Spin`` mode for the ``CSGPolygon`` and draw a trapezoid (American English) while in ``Front View`` (numeric keypad 1); this shape will extrude around the origin and form the lampshade.
```
Comment (LikeLakers2) [inline tutorials/3d/csg_tools.rst]: I'm questioning why we need `(American English)` here.

Comment (corrigentia) [reply tutorials/3d/csg_tools.rst]: > We don't.

@akien-mga : I beg to differ.
tl;dr : I would call the geometrical figure a ``trapezium`` and that had been the word used to describe a quadrilateral with two parallel sides (in this case, the top and bottom) prior to ``trapezoid``.

```
.. image:: img/csg_lamp_polygon.png
```
[https://github.com/godotengine/godot-docs/blob/master/tutorials/3d/img/csg_lamp_polygon.png](https://github.com/godotengine/godot-docs/blob/master/tutorials/3d/img/csg_lamp_polygon.png)
[https://raw.githubusercontent.com/godotengine/godot-docs/master/tutorials/3d/img/csg_lamp_polygon.png](https://raw.githubusercontent.com/godotengine/godot-docs/master/tutorials/3d/img/csg_lamp_polygon.png)
csg_lamp_polygon

[trapezium](https://www.thefreedictionary.com/trapezium) =  **Originally**, a quadrilateral with two  **parallel sides**  (**later confused**  with _trapezoid_, originally, a quadrilateral having no parallel sides), from Modern Latin trapezium, a quadrilateral with two parallel sides [...].

[Trapezium](https://en.wikipedia.org/wiki/Trapezium), describing a geometric shape, has two contradictory meanings:
1.    Outside the US and Canada: a quadrilateral with at least one pair of  **parallel sides**  (known in the US as a trapezoid)
2.    In the US and Canada: a quadrilateral with  **NO**  parallel sides (known elsewhere as a general irregular quadrilateral)

[trapezoid](https://www.thefreedictionary.com/trapezoid) =  **Originally**, a quadrilateral with  **NO**  parallel sides  (**later confused**  with _trapezium_, originally, a quadrilateral with two parallel sides), from New Latin trapezoīdēs, a quadrilateral with no parallel sides [...].

In conclusion, use American English if you must (for wider acceptance), but know when it is in the wrong.

Comment (LikeLakers2) [reply tutorials/3d/csg_tools.rst]: I'm still not sure what the confusion is. If someone doesn't know what a "trapezoid" is, they're likely to google it, and they'll get the definition for "trapezoid", not "trapezium".

Comment (LikeLakers2) [reply tutorials/3d/csg_tools.rst]: First off, I'm not quite sure why it matters. Both still refer to a shape, and this is a tutorial where nobody's going to care if you draw a lampshade or the Eiffel Tower. Making the distinction sounds a little petty.

That said, why don't we just replace "trapezoid" with "half-lampshade shape", or something to that effect? It would probably make this section a lot easier to understand right off the bat, without all the fuss.

Comment (corrigentia) [reply tutorials/3d/csg_tools.rst]: > That said, why don't we just replace "trapezoid" with "half of a lampshade shape", or something to that effect? It would probably make this section a lot easier to understand right off the bat, without all the fuss.

@LikeLakers2 : I'm with you, I don't want to create unnecessary confusion.

> First off, I'm not quite sure why it matters. Both still refer to a shape, plus this is a tutorial -- nobody's going to care if you draw a lampshade or the Eiffel Tower. Making the distinction just seems unnecessary.

I was considering letting readers coming from a British English background know why this particular word had been chosen instead of the obvious one (in this context), trapezium. But if it seems redundant and out of place to at one least one person (you), then I agree that note should be removed.

**Comment by akien-mga** [comment]
Thanks!
